content/2023/kickstart.rst

```diff
@@ -0,0 +1,85 @@
+why, and why you might want to attend.
+
+The full name is "Introduction to scientific computing and HPC"
+(high-performance computing), and it used to be called "HPC Kickstart"
+and was taught without the first day, thus the short name "kickstart"
+we still use.  Some years day 1 had a different name, but was still
+taught together with days 2-3 as a package.
+
+
+
+Topics of SciComp Kickstart
+---------------------------
+
+**Day 1** is not about high-performance computing things, but the
+basic skills needed to do scientific computing: things like Linux
+usage, data management, the types of tools available for different
+problems.  For almost anyone doing any kind of programming/scientific
+computing kind of work, regardless of background.  These kind of
+skills aren't taught in academic degree programs.  We teach these on
+day 1 because otherwise, new researchers have to learn from each other
+or re-invent.
+
+**Days 2 and 3 are about high-performance computing, more precisely
+basic cluster usage** (with a focus of the basics).  This is focused
+on the kinds of tools our community usually uses.
+
+
+The topics are refined after many years of both teaching and support
+of junior researchers.  Because of the way academic careers work (much
+diversity of paths), these topics (even day 1) aren't just for new
+researchers but everyone can find something to learn or brush up on.
+
+
+
+Yearly schedule
+---------------
+
+For the past years, we have been trying to keep up this yearly summer
+schedule.  This usually happens the first full workweek:
+
+* **Monday:** HR introductions, other formalities for new summer
+  workers - many departments seem to something like this.  This may
+  happen early than Monday of the kickstart week, since sometimes that
+  comes too late.
+* **Tuesday afternoon:** Kickstart course day 1, the general scientific
+  computing introduction.  Applicable to everyone doing scientific
+  computing.
+* **Wednesday-Thursday afternoons:** The HPC cluster usage part, which fewer
+  people will attend compared to Tuesday.
+* **Friday:** we don't have scheduled programs on Fridays, but
+  sometimes there are communities who host advanced tutorials here
+  about what their local users need.  In 2023, there is at least an
+  advanced GPU course then.
+
+
+
+Future
+------
+
+In the past, we have tried to coordinate with the department HRs to
+avoid any other events, and have been mostly successful.  In 2023, we
+tried extra hard to do this coordination early and thought we were
+successful.  Unfortunately, the CS department scheduled the summer
+event on Tuesday of the week we have planned.  There have been similar
+things in the past and we have adjusted our schedule, but
+unfortunately this year, with everyone else coordinated, we have no
+option to do that.  **This places CS researchers in an unfortunate
+position of choosing between skill and networking for career
+development**.
```

Comment:
```suggestion
This years scheduling conflict
------------------------------

We just noticed that there is a scheduling conflict with the CS summer day 
which was announced to be on the Tuesday of the HPC kickstart course. We 
apologize that this creates an inconvenience for CS members who want to attend
both events, but we unfortunately can no longer reschedule the course since it 
was also coordinated with several other departments. We will do our best to 
avoid these kinds of conflicts in the future. 
```

Reply:
We were just talking about it and admittedly the only way to not put blame or accept blame is simply to NOT mention it. 
If people notice and ask, we can always say what happened, but other than that....